kbs/src/plugins/implementations/splitapi/backend.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2025 by IBM Corporation
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use serde::Deserialize;
+use std::{path::PathBuf, sync::Arc};
+
+use super::generator::CertificateDetails;
+use super::manager;
+
+pub const CREDENTIALS_BLOB_FILE: &str = "certificates.json";
```

Comment:
You're using the term `credential` throughout the PR, but also the term certificates. It might be clearer to just use one especially on lines like this one.